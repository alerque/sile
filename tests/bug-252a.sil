\begin[papersize=a7,class=book]{document}
\use[module=packages.lorem]
\use[module=packages.rebox]
\footnote:separator{\skip[height=5pt]}
\define[command=deepbox]{\rebox[depth=60pt,height=10pt]{x}}
This is a test\footnote{\deepbox}.
\end{document}