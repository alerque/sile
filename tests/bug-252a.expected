Set paper size 	209.7637818	297.6377985
Begin page
Mx 	49.3669
My 	42.0943
Set font 	Gentium Plus;10;400;;normal;;LTR
T	55 75 76 86 w=17.5977	(This)
Mx 	69.6166
T	76 86 w=6.5723	(is)
Mx 	78.8408
T	68 w=4.5898	(a)
Mx 	86.0826
T	87 72 86 87 w=15.3662	(test)
Mx 	101.4489
My 	38.9156
Set font 	Gentium Plus;6.81152;400;;normal;;LTR
T	20 w=3.1962	(1)
Mx 	104.6451
My 	42.0943
Set font 	Gentium Plus;10;400;;normal;;LTR
T	17 w=2.2900	(.)
Mx 	29.3669
My 	187.9323
Set font 	Gentium Plus;9;400;;normal;;LTR
T	20 w=4.2231	(1)
Mx 	33.5901
T	17 w=2.0610	(.)
Mx 	53.6511
T	91 w=4.5264	(x)
Mx 	108.5140
My 	262.9405
Set font 	Gentium Plus;10;400;;normal;;LTR
T	20 w=4.6924	(1)
End page
Finish
